hops/util.py

```diff
@@ -496,3 +506,23 @@ def get_jwt():
     """
     with open(constants.REST_CONFIG.JWT_TOKEN, "r") as jwt:
         return jwt.read()
+
+def get_api_key_aws(project_name):
```

Comment:
@steffengr we can make this method more generic to support multiple cloud providers, either by having a `cloud` variable which can accepts values such as `aws`, `gcp`, `azure` etc. and then connect accordingly or by having a `Cloud` and then extend it for every cloud provider?

Reply:
That will be an easy refactoring as soon as we want to support a second provider and we should defer this until we have a concrete need.

Reply:
Then we need to at least remove the hard-coded region.

Reply:
Right totally missed that. Thanks!

Reply:
By the way, Hopsworks already has a `cloud` variable set in karamel-chef cluster definition
https://github.com/logicalclocks/hopsworks/blob/44561f393d8d848545c30cc0cbe91207bfbf20d5/hopsworks-common/src/main/java/io/hops/hopsworks/common/util/Settings.java#L3456

Reply:
Yes but this will not be used together with chef as it's for SageMaker only. I believe that we will need separate clients per platform anyways in the future as this one is getting bloated and lot's of the APIs are not useful in every context. 
For the region,  I'll simply remove it for now and it will assume the default which in SageMaker's case is the region its instance is running in which seems a reasonable requirement for now and avoids passing it around everywhere in the code.